sos/archive.py

```diff
@@ -342,12 +342,14 @@ def add_file(self, src, dest=None):
                 # path case
                 try:
                     shutil.copy(src, dest)
-                except IOError as e:
+                except (IOError, OSError) as e:
                     # Filter out IO errors on virtual file systems.
                     if src.startswith("/sys/") or src.startswith("/proc/"):
                         pass
                     else:
                         self.log_info("caught '%s' copying '%s'" % (e, src))
+                    if e.errno == 2:
+                        self.log_info("File not collected: '%s' - Reason: '%s'" % (src, e))
```

Comment:
This would lead to messages like:

```
2019-12-05 14:20:50,159 INFO: [sos.archive:add_file] File not collected: '/path/not/found' - Reason: '[Errno 2] No such file or directory: '/path/not/found'
```

Which seems a bit longwinded/redundant. It's fine to just use the exception string as the message since it already includes the path.

Reply:
Removed redundant message from log